<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1102</width>
    <height>736</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="my_label" name="frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>700</width>
      <height>700</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>CrossCursor</cursorShape>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="mouse_movement">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>40</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>10</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="text">
     <string>      Mouse Movement</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
   </widget>
   <widget class="QLabel" name="mouse_pressed">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>40</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>10</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>  Mouse Pressed</string>
    </property>
   </widget>
   <widget class="Line" name="x_axis">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>225</y>
      <width>700</width>
      <height>1</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 85, 255);</string>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="y_axis">
    <property name="geometry">
     <rect>
      <x>225</x>
      <y>0</y>
      <width>1</width>
      <height>700</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
background-color: rgb(85, 255, 255);</string>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>180</y>
      <width>391</width>
      <height>151</height>
     </rect>
    </property>
    <property name="title">
     <string>Line </string>
    </property>
    <widget class="QPushButton" name="set_point1">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>111</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>First Point</string>
     </property>
    </widget>
    <widget class="QPushButton" name="set_point2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>111</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Second Point</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="draw_line">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>111</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Draw Line</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_5">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>20</y>
       <width>120</width>
       <height>131</height>
      </rect>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QRadioButton" name="bresenham">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>99</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Bresenham</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="dda">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>99</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>DDA</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Time in ms:</string>
      </property>
     </widget>
     <widget class="QLabel" name="time_box">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>530</y>
      <width>391</width>
      <height>121</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QSpinBox" name="grid_size">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>10</y>
       <width>42</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QRadioButton" name="draw_grid">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>99</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Draw Grid</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Draw">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>60</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Draw</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>60</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>RESET</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="interpolate">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>10</y>
       <width>101</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Interpolate</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>70</y>
      <width>391</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string>Circle</string>
    </property>
    <widget class="QRadioButton" name="draw_circle">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>101</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Draw Circle</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="circle_radius">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>30</y>
       <width>46</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>20</y>
       <width>120</width>
       <height>91</height>
      </rect>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QRadioButton" name="midpoint_cirlce">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>99</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Mid-Point</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="bresenham_circle">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>99</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Bresenham</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="polar_circle">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>99</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Polar</string>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>80</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Time in ms:</string>
     </property>
    </widget>
    <widget class="QLabel" name="time_box_3">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>80</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QCheckBox" name="show_axes">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>10</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Show Axes</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_6">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>330</y>
      <width>391</width>
      <height>101</height>
     </rect>
    </property>
    <property name="title">
     <string>Ellipse</string>
    </property>
    <widget class="QRadioButton" name="draw_ellipse">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>99</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Draw Ellipse</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="ellipse_a">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>42</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QSpinBox" name="ellipse_b">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>60</y>
       <width>42</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_7">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>20</y>
       <width>120</width>
       <height>80</height>
      </rect>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QRadioButton" name="midpoint_ellipse">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>99</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Mid-Point</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="polar_ellipse">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>99</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Polar</string>
      </property>
     </widget>
     <widget class="QLabel" name="time_box_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>60</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Time in ms:</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_8">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>430</y>
      <width>391</width>
      <height>91</height>
     </rect>
    </property>
    <property name="title">
     <string>Polygon</string>
    </property>
    <widget class="QRadioButton" name="draw_polygon">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>121</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Draw Polygon</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_9">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>20</y>
       <width>131</width>
       <height>71</height>
      </rect>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QRadioButton" name="scanline">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>101</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Scanline Fill</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="flood">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>99</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Flood Fill</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="boundary">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>99</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Boundary Fill</string>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="fill_button">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>40</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>FILL</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1102</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>my_label</class>
   <extends>QLabel</extends>
   <header location="global">my_label.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
